packages/types/api/providers.d.ts

```diff
@@ -0,0 +1,32 @@
+export interface ProviderRecord {
```

Comment:
Also, I had forgotten that we had any types in this package, but I had been defining all new RDS types in the `db` package. I kind of like them all being in the `db` package, but most importantly I dislike them being in multiple places. So we should settle on one or the other

Reply:
It seems to be already defined here https://github.com/nasa/cumulus/blob/feature-rds/packages/db/src/types.ts#L40-L53

Reply:
@botanical  @markdboyd  That's most definitely the correct type for PostgresProviderRecord, and I agree that should be in the db package - I'll update the reference and remove the dupe here, thanks for the feedback.   I'll leave the 'dynamo' provider record here.

Reply:
FWIW - this type is not the same as the one in the `db` package. For example, the `created_at` field on the type in the `db` package is required, but optional here:

https://github.com/nasa/cumulus/blob/feature-rds/packages/db/src/types.ts#L51

So is this type just unnecessary given the one we have for insertion data below, `PostgresProviderRecord`?

Reply:
@markdboyd  I'm not entirely sure what you're getting at there, prior to the updates both types were in that file, and the Postgres typing was most definitely a direct dupe, and there was a correct API provider.  

In anycase, it's been updated: c591c12ea